tasks/cockpit-tasks

```diff
@@ -80,3 +80,8 @@ done
 # Prune old images
 update_bots
 ./image-prune
+
+# Clean up S3
+for region in eu-central-1 us-east-1; do
+    ./image-prune --s3 "https://cockpit-images.${region}.linodeobjects.com/" || true
```

Comment:
Is that alternative truth just to make the integration tests happy? TBH I'd rather get a "bot crashed" notification if pruning fails. If it's just for the tests, could that rather be guarded by some `[ -e the_secret ]`?

Reply:
I was actually thinking about network issues...

Reply:
...but now that you mention it, I've heard that it's better to beg forgiveness than to ask permission.